src/Microsoft.AspNetCore.Server.Kestrel/Http/Frame.cs

```diff
@@ -118,6 +119,7 @@ public string HttpVersion
         public IHeaderDictionary RequestHeaders { get; set; }
         public Stream RequestBody { get; set; }
 
+        public bool RequestRejected => _requestRejectionReason != RequestRejectionReason.None;
```

Comment:
The previous field `_corruptedRequest` was `protected` rather than `public`.  Should `RequestRejected` also be protected?